scenarios/hypertension/hypertension_contactable_overdue_patients_called_and_returned_to_care.feature

```diff
@@ -0,0 +1,315 @@
+      | Address (current)                     | Rose Gardens |
+      | Patient Phone Number                  | 345672781624 |
+      | District                              | KOLARA       |
+      | HTN - Consent to record data          | true         |
+      | HTN - NCD Patient Status              | ACTIVE       |
+    And That TEI has a "Hypertension & Diabetes visit" event on "7_MonthsAgo" with following data
+      | Systole  | 142 |
+      | Diastole | 95  |
+    And That TEI has a "Hypertension & Diabetes visit" event scheduled for "6_MonthsAgo"
+    And That TEI has a "Calling report" event on "5_MonthsAgo_Plus_1_Day" with following data
+      | Result of call                          | AGREE_TO_VISIT |
+    And That TEI has a "Hypertension & Diabetes visit" event on "5_MonthsAgo_Plus_10_Day" with following data
+      | Systole  | 142 |
+      | Diastole | 95  |
+
+    When I wait for 1 second
+    And I export the analytics
+
+    Then The value of "PI":"HTN - Contactable overdue patients called and returned to care" with period type "Months" should be
+      | 4_MonthsAgo  | 0 |
+      | 5_MonthsAgo  | 1 |
+      | 6_MonthsAgo  | 0 |
+
+  Scenario: Overdue non-contactable patients called returned to care
+    Given I am signed in as a user with role "Superuser"
+    And I have access to an organisation unit at level 5
+    And I register that organisation unit for program "Hypertension & Diabetes"
+
+    Given I create a new TEI on "7_MonthsAgo" for this Facility with the following attributes
+      | GEN - Given name                      | Priyanka     |
+      | GEN - Family name                     | Chopra       |
+      | GEN - Sex                             | MALE         |
+      | HTN - Does patient have hypertension? | YES          |
+      | HTN - Does patient have diabetes?     | YES          |
+      | GEN - Date of birth                   | 32           |
+      | Address (current)                     | Rose Gardens |
+      | District                              | KOLARA       |
+      | HTN - Consent to record data          | true         |
+      | HTN - NCD Patient Status              | ACTIVE       |
+    And That TEI has a "Hypertension & Diabetes visit" event on "7_MonthsAgo" with following data
+      | Systole  | 142 |
+      | Diastole | 95  |
+    And That TEI has a "Hypertension & Diabetes visit" event scheduled for "6_MonthsAgo"
+    And That TEI has a "Calling report" event on "5_MonthsAgo_Plus_1_Day" with following data
+      | Result of call                          | AGREE_TO_VISIT |
+    And That TEI has a "Hypertension & Diabetes visit" event on "5_MonthsAgo_Plus_10_Day" with following data
+      | Systole  | 142 |
+      | Diastole | 95  |
+
+    When I wait for 1 second
+    And I export the analytics
+
+    Then The value of "PI":"HTN - Contactable overdue patients called and returned to care" with period type "Months" should be
+      | 4_MonthsAgo  | 0 |
+      | 5_MonthsAgo  | 0 |
```

Comment:
I could be wrong, but shouldn't this patient appear as overdue in 5_MonthsAgo report, as it had a scheduled visit for 6_MonthsAgo but there was no visit until 5_MonthsAgo_Plus_10_Day. The calling report event happens on 5_MonthsAgo_Plus_1_Day so the patient must have appeared as overdue by then ?